### DISCUSSION TITLE
Vulnerability of dependency "golang.org/x/net, golang.org/x/text"

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [IBAX-io/go-ibax#2126] [Ideas] [answered]
Hello, we are a team researching the dependency management mechanism of Golang. During our analysis, we came across your project and noticed that it contains a vulnerability (CVE-2022-41723, CVE-2022-41717, CVE-2022-27664, CVE-2022-32149). 

In your project, the golang.org/x/net package is being used at version golang.org/x/net v0.0.0-20220722155237-a158d28d115b, but the patched version is v0.7.0. To fix the vulnerability, we recommend modifying the go.mod file to update the version to v0.7.0 or higher. 

the golang.org/x/text package is being used at version golang.org/x/text v0.3.7, but the patched version is v0.3.8. To fix the vulnerability, we recommend modifying the go.mod file to update the version to v0.3.8 or higher. 

Thank you for your attention to this matter.

**Comment by yolandadadada** [comment, marked as answer]
Thank you for your attention to IBAX and for your technical suggestions for us, after seeing your message we fixed the vulnerability at the first time, in order to thank you for your contribution, you can contact @yolandadadada on telegram, we give some IBXC coins as a reward.

**Comment by zeland2323** [comment]
Thanks
